proxy/proxyPriv_proxy.go

```diff
@@ -268,7 +268,14 @@ func (mgr *proxyPrv) listen() (net.Listener, error) {
 		logger.Warningf("[%s] tcp listener get file failed, err: %v", err)
 		return nil, err
 	}
-	defer file.Close()
+
+	// defer file.Close()
+	defer func(){
+		if err := file.Close(); err != nil{
+			panic(err)
```

Comment:
Why panic? Just log to error level should be fine.

Reply:
ok，I just donit know what to log~